gems/smithy/lib/smithy/vise/service_shape.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+module Smithy
+  module Vise
+    # Represents a service shape from the model.
+    class ServiceShape < Shape
+      # (see Shape#initialize)
+      def initialize(id, shape)
+        super
```

Comment:
I didn't know if there was a standard to have `super` first or after attributes. 
I think I'll follow this format (and update what I have going on.

Reply:
It depends on what the superclass or in most cases it does not matter. I prefer it first if functionality is added and last if values set by the subclass need to influence the super class.